frontend/src/lib/services/Libre311/Libre311.ts

```diff
@@ -265,6 +271,11 @@ export type ServiceRequest = z.infer<typeof ServiceRequestSchema>;
 export const GetServiceRequestsResponseSchema = z.array(ServiceRequestSchema);
 export type GetServiceRequestsResponse = z.infer<typeof GetServiceRequestsResponseSchema>;
 
+// ***************** Create Group ***************** //
+export const CreateGroupParamsSchema = z.object({ name: z.string() });
```

Comment:
For future reference, you don't have to use zod for internal types.  We just want to use zod to ensure that the objects that we are receiving from the server are as we expect them to be.  That said, it isn't wrong so don't worry about updating it.